sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/PreDefinedTagsOperations.cs

```diff
@@ -0,0 +1,163 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+
+namespace Azure.ResourceManager.Core
+{
+    /// <summary> The Tags service client. </summary>
+    public class PreDefinedTagsOperations : OperationsBase
+    {
+        /// <summary>
+        /// The resource type for Tags.
+        /// </summary>
+        public static readonly ResourceType ResourceType = "Microsoft.Resources/subscriptions/tagNames";
```

Comment:
shouldn't this be "Microsoft.Resources/tagNames"?

Reply:
Done